### DISCUSSION TITLE
uploads and downloads in log

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#334] [Show and tell]
in 0.48.0 you'll get this new "notes" column in the Logs page, where you get important information about downloads (when they are completed) and uploads (the size).
<img width="850" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/78465250-ef94-4289-a6d8-1414ea742718">

Another important improvement is that, in case of upload, the URI will display the full path of the upload, instead of just the folder.

It will also be possible to filter by notes
<img width="663" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/d8fa33c4-8e55-4a29-8c0a-8625b69c5502">

**Comment by Athep** [comment]
Thank you for adding this. But it seems there is some issue. The "fully downloaded" note in the log is added as soon as a file is accessed. It says completed even if 0 byte is downloaded. I tested this with a 500 MB file.

**Comment by rejetto** [comment]
thank you for your report @Athep !
next release will fix this

**Comment by rejetto** [comment]
https://github.com/rejetto/hfs/releases/tag/v0.48.0-alpha3
